internal/corazawaf/transaction.go

```diff
@@ -776,7 +776,7 @@ func (tx *Transaction) ProcessRequestBody() (*types.Interruption, error) {
 		return nil, err
 	}
 
-	// Chunked requests will always be written to a temporary file
```

Comment:
I think this is important and makes total sense. Is this still the case?

Reply:
Thanks for the research! I misread the comment when removing it, but coincidentally it should be okay to remove it